apps/web/src/layouts/LayoutWrapper.tsx

```diff
@@ -59,7 +45,7 @@ export function LayoutWrapper({ children }: { children: ReactNode }) {
   }
 
   //@ts-ignore
-  if (isBlocked(signer?._address))
+  if (isBlocked(address))
```

Comment:
Do we still need that TS ignore? 

Looks like the cause of the problem before was that _address wasn't found on type signer.